src/App.vue

```diff
@@ -22,15 +23,29 @@ import { Todo } from '@/types/Todo'
 const todoList = ref<Todo[]>([])
 const isTodoListEmpty = computed(() => !todoList.value.length)
 
+const counter = ref(0)
 // Function that simulates a todo add for testing
 function addTodo() {
   const newItem: Todo = {
-    id: todoList.value.length,
+    id: counter.value++,
     priority: 'Low',
-    title: 'Test Title',
-    description: 'This is a test description to check the',
-    isChecked: false
+    title: 'New Todo',
+    description: 'This is a placeholder text. Click on the card to start editing!',
+    isChecked: false,
+    date: getDate()
   }
   todoList.value.unshift(newItem)
 }
+
+function getDate() {
```

Comment:
This date generation is great. But if you want to use package which is a more user-friendly try [this one](https://momentjs.com/)